app/helpers/application_helper/toolbar/chargeback_rate_center.rb

```diff
@@ -20,14 +20,14 @@ class ApplicationHelper::Toolbar::ChargebackRateCenter < ApplicationHelper::Tool
           t = N_('Copy this Chargeback Rate'),
           t,
           :klass        => ApplicationHelper::Button::ChargebackRates,
-          :url_parms    => "main_div",
+          :url          => '/copy',
           :send_checked => true),
         button(
           :chargeback_rates_delete,
           'pficon pficon-delete fa-lg',
           N_('Remove this Chargeback Rate from the VMDB'),
           N_('Remove from the VMDB'),
-          :url_parms    => "main_div",
+          :url          => '/delete',
```

Comment:
I'm pretty sure we can do somehow a POST request that doesn't want to evaluate the (nonexistent) `button` method.

Reply:
@skateman changed `delete` button to send up a `POST` request